style.css

```diff
@@ -1,645 +1,732 @@
+  transform: scale(0);
+  background: rgba(193, 199, 208, 0.5);
+  z-index: 200;
+}
+
+.overlay.active {
+  visibility: visible;
+  transform: scale(1);
+  opacity: 1;
+  pointer-events: all;
+}
+
+.navigation_bar__menu__link {
+  transition: color 0.5s ease-in-out, font-size 0.5s ease-in-out;
+}
+
+@media (min-width: 992px) {
+  .navigation_bar__menu__link:hover {
+    color: #396df2;
+    font-size: 17px;
+  }
+}
+
+.icons_list__link img {
+  transition: transform 0.5s;
+}
+
+.about__icons__link img {
+  transition: transform 0.5s;
+}
+
+.icons_list__link img:hover {
+  transform: scale(1.5);
+}
+
+.about__icons__link img:hover {
+  transform: scale(1.5);
+}
+
+@media (max-width: 992px) {
+  .navigation_bar__menu {
+    visibility: hidden;
+    opacity: 0;
+    width: 100%;
+    height: 100vh;
+    background: rgba(96, 112, 255, 0.95);
+    display: flex;
+    position: fixed;
+    top: 0;
+    left: 100%;
+    transition: left 0.8s, visibility 0.8s, opacity 0.4s;
+    padding: 8rem 2rem;
+    z-index: 50;
+  }
+
+  .navigation_bar__menu__ul {
+    flex-direction: column;
+  }
+
+  .navigation_bar__menu__li {
+    margin-bottom: 40px;
+  }
+
+  .navigation_bar__menu__li a {
+    font-size: 32px;
+    font-weight: 600;
+    line-height: 44px;
+    color: #fff;
+    transition: color 1s;
+  }
+
+  .navigation_bar__menu__li a:hover {
+    color: #000;
+  }
+}
+
+.active {
+  visibility: visible;
+  opacity: 1;
+  left: 0;
+  transition: left 0.8s, visibility 0.8s, opacity 0.4s;
+}
+
+@media (min-width: 992px) {
+  .navigation_bar {
+    padding: 1.5rem 8rem;
+  }
+
+  .navigation_bar__menu {
+    display: flex;
+    align-items: center;
+  }
+
+  .navigation_bar__menu__li {
+    margin: 0 16px;
+  }
+
+  .head::before {
+    padding: 6rem;
+    border-bottom-left-radius: 8rem;
+  }
+
+  .head {
+    padding: 10rem;
+    padding-right: 15rem;
+  }
+
+  .head__title span {
+    width: 15ch;
+  }
+
+  .head__link {
+    margin: 0.7rem 0;
+  }
+
+  .cards {
+    padding: 5rem;
+  }
+
+  .card {
+    align-items: center;
+    max-width: 1104px;
+    column-gap: 46px;
+    padding: 1.5rem;
+  }
+
+  .card__text {
+    padding: 1rem;
+  }
+
+  .card__img {
+    width: 50%;
+  }
+
+  .card:nth-child(1),
+  .card:nth-child(3) {
+    flex-direction: row;
+  }
+
+  .card:nth-child(2),
+  .card:nth-child(4) {
+    flex-direction: row-reverse;
+  }
+
+  .about {
+    padding: 6rem;
+    flex-direction: row;
+    column-gap: 37px;
+    border-top-right-radius: 8rem;
+  }
+
+  .about_me {
+    flex-basis: 25rem;
+  }
+
+  .langs__bottom {
+    display: flex;
+    column-gap: 12px;
+  }
+
+  .langs__bottom li {
+    flex-direction: column;
+    align-items: flex-start;
+    flex-basis: 7.6rem;
+  }
+
+  .tools {
+    flex-basis: 25rem;
+  }
+}
+
+.form-container-2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  border-top-left-radius: 10%;

```

Comment:
You seem to be missing out a little detail about the corner radius of the section. Kindly ensure that it's as close to the design as possible.

Also, the background image of the section is missing

Your design:
![image](https://user-images.githubusercontent.com/20554583/230055625-e8215355-1cf6-47e3-a844-e4186d995195.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/230056785-9151efff-5308-4574-ba68-df15db165b42.png)

Reply:
🆗 Thank you for your nice coment